src/components/notifications/Notifications.tsx

```diff
@@ -291,7 +291,7 @@ const Notifications = ({
 
   const panProps = canShowNotifications ? panResponder.panHandlers : {}
 
-  return (
+  return isSignedIn ? (
```

Comment:
is this true when the session first loads? (i think probably, but just want to make sure notifs are accessible right away still)

Reply:
yes it is